static/app/components/discover/performanceCardTable.tsx

```diff
@@ -0,0 +1,667 @@
+import {Fragment} from 'react';
+import * as React from 'react';
+import {Link} from 'react-router';
+import styled from '@emotion/styled';
+import {Location, LocationDescriptor, Query} from 'history';
+
+import Alert from 'app/components/alert';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {PanelItem} from 'app/components/panels';
+import PanelTable from 'app/components/panels/panelTable';
+import {backend, frontend, mobile, serverless} from 'app/data/platformCategories';
+import {IconArrow, IconWarning} from 'app/icons';
+import {t} from 'app/locale';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {Organization, ReleaseProject} from 'app/types';
+import {TableData, TableDataRow} from 'app/utils/discover/discoverQuery';
+import EventView, {MetaType} from 'app/utils/discover/eventView';
+import {getFieldRenderer} from 'app/utils/discover/fieldRenderers';
+import {getAggregateAlias} from 'app/utils/discover/fields';
+import {getDuration} from 'app/utils/formatters';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+import {GridCell, GridCellNumber} from 'app/views/performance/styles';
+import {TrendsDataEvents} from 'app/views/performance/trends/types';
+import {DiscoverField} from 'app/views/releases/detail/overview/index';
+import {performanceCardLabels} from 'app/views/releases/detail/utils';
+
+const FRONTEND_PLATFORMS: string[] = [...frontend, ...mobile];
+const BACKEND_PLATFORMS: string[] = [...backend, ...serverless];
```

Comment:
Based on designs, should there be a MOBILE_PLATFORMS section?
https://www.figma.com/file/jsZJ9v8ditpm4uBISafH9P/Releases-V4?node-id=710%3A11400

Reply:
Ah yeah there is, for some reason I couldn't find how to get that data, but I found the files - just need to see what it already looks like in the product